Sunrise/Models/Attribute.swift

```diff
@@ -20,4 +20,27 @@ struct Attribute: Mappable {
         value              <- map["value"]
     }
 
+    // MARK: - Value string representation
+
+    func value(type: AttributeType) -> String? {
```

Comment:
@cneijenhuis - do you think we should support any of the other `AttributeType`s at this point? If so, should we support all of them, except `NestedType` which is in beta, or you have some subset on your mind that we should add right away?